### DISCUSSION TITLE
Vivado installation problem

### DISCUSSION THREAD

**Discussion by Markkkk1218** [ichi4096/vivado-on-silicon-mac#57] [Q&A]
As I follow the steps. The terminal shows the following instructions:
''You need to put the Vivado installation file into this folder if you have not done so already.
Then, drag and drop the Vivado installation binary into this terminal window and press Enter: ''

Here's the question:
What and where is the Vivado installation file?
Put into what folder?
Where is the Vivado installation binary?

Thanks!

**Comment by ichi4096** [comment]
Just follow the steps in the "How to install" section on this repository's main page: You need to download the Vivado installer from Xilinx' website and put it inside the folder from which You run the scripts. Then drag the installer into the terminal window and press enter.

**Reply by Markkkk1218** [reply]
Thanks! 
After Vivado was successfully installed. I run ''Downloads/vivado-on-silicon-mac-main/scripts/start_container.sh'' and have problem that the Screen Sharing APP shows ''Failed to connect the localhost''. 
Thank you!
![截圖 2024-10-17 13 57 47（2）](https://github.com/user-attachments/assets/a81eb354-235c-426b-a322-88ce42c8c7a4)

**Reply by ichi4096** [reply]
"VMAllocationTracker.cpp" errors mean that the emulation doesn't work, which I have experienced on macOS 14. You need to be running macOS 15.
